django_project/core/models/collection/models.py

```diff
@@ -0,0 +1,16 @@
+from django.db import models
+from core.models.metadata.models import Metadata
+
+
+class CollectionType(Metadata):
+    name = models.CharField(max_length=128)  # 'farmed', 'wild-caught', etc.
+
+    def __str__(self):
+        return self.name
+
+
+class CollectionMethod(Metadata):
```

Comment:
Are all collection methods a subset of wild-caught? If so, maybe they should be one field, and there should be multiple types of wild-caught?

Reply:
I wanted to leave it open to other categories. For instance, there are multiple methods of fish farming (https://en.wikipedia.org/wiki/Fish_farming) with their respective ecological/ethical pros/cons.

However, it still may make sense to have them be one field, where instead of 'broad'/'specific' pairings  we just skip straight to specific (e.g. 'farmed'/'intensive' -> 'intensive farming'). Some of the techniques are so dramatically different that even though they're the same CollectionType it doesn't really make sense to draw equivalence between them.